Cargo.toml

```diff
@@ -18,19 +18,19 @@ futures = ["std", "futures-io", "futures-util"]
 futures-codec = ["std", "bytes", "futures_codec"]
 
 [dependencies]
-bytes = { version = "0.5.3", optional = true }
-futures-io = { version = "0.3.4", optional = true }
-futures-util = { version = "0.3.4", features = ["io"], optional = true }
-futures_codec = { version = "0.4", optional = true }
-tokio-util = { version = "0.3.1", features = ["codec"], optional = true }
-nom = { version = "5", optional = true }
+bytes = { version = "0.5.6", optional = true }
```

Comment:
```suggestion
bytes = { version = "0.6.0", optional = true }
```

Reply:
I recall there being a conflict with this, and I had to specifically downgrade it. 

Does this update still compile? If so, lets add it in